cpp/turbodbc_numpy/Library/src/numpy_result_set.cpp

```diff
@@ -48,10 +48,15 @@ numpy_result_set::numpy_result_set(turbodbc::result_sets::result_set & base) :
 {
 }
 
-
-pybind11::object numpy_result_set::fetch_all()
+pybind11::object numpy_result_set::fetch_next_batch()
 {
 	std::size_t rows_in_batch = base_result_.fetch_next_batch();
+
+	if (!rows_in_batch)
```

Comment:
It is not a C++11 thing, it's probably just my thing. I also don't like `&&` (use `and`) or `||` (use `or`). I'm just glad that Python got this part right ;-).